### DISCUSSION TITLE
2D only fork

### DISCUSSION THREAD

**Discussion by boruok** [godotengine/godot-proposals#8751] [General Discussions]
is here any 2d-only Godot fork? wanna know theoretical min size of templates.

**Comment by AThousandShips** [comment]
You can just build with no 3d, see the [instructions](https://docs.godotengine.org/en/stable/contributing/development/compiling/optimizing_for_size.html#disabling-3d)

**Comment by saierXP** [comment]
Here https://github.com/WhalesState/pixel-engine
